src/table/styles.js

```diff
@@ -0,0 +1,135 @@
+import { styled, Paper, TableContainer, TableRow, TableSortLabel, TableBody, Select, IconButton } from '@mui/material';
+
+// ---------- AnnounceWrapper ----------
+
+export const TableAnnouncer = styled('div')({
+  clip: 'rect(0 0 0 0)',
+  clipPath: 'inset(50%)',
+  height: '1px',
+  overflow: 'hidden',
+  position: 'absolute',
+  whiteSpace: 'nowrap',
+  width: '1px',
+});
+
+// ---------- FooterWrapper ----------
+
+export const StyledFooterWrapper = styled(Paper, {
+  shouldForwardProp: (prop) => prop !== 'tableTheme',
```

Comment:
should be `shouldForwardProp: (prop) => prop !== 'tableTheme' && prop !== 'theme', `?

Reply:
no theme is the mui theme, no need to suppress it here

Reply:
right, I saw it is used by the component itself